fe/fe-core/src/main/java/com/starrocks/common/FeMetaVersion.java

```diff
@@ -199,6 +199,8 @@
     public static final int VERSION_91 = 91;
     // for mysql external table support resource
     public static final int VERSION_92 = 92;
+    // for load statics
+    public static final int VERSION_93 = 93;
     // note: when increment meta version, should assign the latest version to VERSION_CURRENT
-    public static final int VERSION_CURRENT = VERSION_92;
+    public static final int VERSION_CURRENT = VERSION_93;
```

Comment:
Actually, changing this version will cause FE can't fallback to the previous version. In order to give users the ability to fallback, starrocks don't change version.
So I suggest you should find a way that support this feature with out changing this version.

Reply:
@imay It's right. But StarRocks seem only support forword compatibility by increase `VERSION_CURRENT `. Do you have any suggestion about this?